Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/parser/CommentParser.java

```diff
@@ -175,13 +200,52 @@ private CharSequence handleAnchor(
             PostLinkable pl = new PostLinkable(theme, handlerLink.key, handlerLink.value, handlerLink.type);
             res.setSpan(pl, 0, res.length(), (250 << Spanned.SPAN_PRIORITY_SHIFT) & Spanned.SPAN_PRIORITY);
             post.addLinkable(pl);
+            spannableStringBuilder.append(res);
 
-            return res;
+            return spannableStringBuilder;
         } else {
             return null;
         }
     }
 
+    private void addMockReply(
+            Theme theme,
+            Post.Builder post,
+            SpannableStringBuilder spannableStringBuilder,
+            int postNo,
+            MockReplyManager.MockReply mockReply
+    ) {
+        Logger.d(TAG, "Adding a new mock reply (replyTo: " + mockReply.getPostNo() + ", replyFrom: " + postNo + ")");
+        post.addReplyTo(mockReply.getPostNo());
+
+        CharSequence replyText = String.format(
```

Comment:
So uh, why even bother with a format string here? Could just be `">>" + mockReply.getPostNo() + " (MOCK)"`. Seems overcomplicated for a dev feature.

Reply:
What the hell. I always thought that String.format() is faster than string concatenation.

Reply:
Done.